pkg/services/rendering/capabilities.go

```diff
@@ -55,3 +57,16 @@ func (rs *RenderingService) HasCapability(ctx context.Context, capability Capabi
 
 	return CapabilitySupportRequestResult{IsSupported: compiledSemverConstraint.Check(compiledImageRendererVersion), SemverConstraint: semverConstraint}, nil
 }
+
+func (rs *RenderingService) IsCapabilitySupported(ctx context.Context, capabilityName CapabilityName) error {
```

Comment:
From a function with an `Is` prefix I would expect a bool as return. Why are we treating `notSupported` as an error? Wouldn't be better to add a bool return (bool, error)and to have different treatments between not supported and an error?

Reply:
The thing is that the `notSupported` error also contains the version from which it's supported.